src/components/Accordion/AccordionHeader.module.css

```
@@ -31,12 +52,19 @@
   border-right-style: var(
     --component-accordion_header_title-border_right_style
   );
+  text-align: var(--component-accordion_header_title-text-align);
   background-color: var(--component-accordion_header_title-color-background);
   font-size: var(--component-accordion_header_title-font_size);
   font-weight: var(--component-accordion_header_title-font_weight);
   margin-left: var(--component-accordion_header_title-spacing-margin_left);
 }
 
 .accordion-header__actions {
-  margin-top: 0.3rem;
+  margin: 0.3rem;
+}
+
+.accordion-header__actions > * {
```

I would instead suggest to target the actions div directly. Targeting `*` will effectively "modify the outside world", and can have unintended consequences. If the child of this element sets f.ex 

```css
display: flex;
gap: 2rem;
```

those rules may not be applied (depends on how strong that selector is), and thus break the layout of the component.


If we instead add these rules directly to the `actions` element, we have full control over those styles, and they will be correctly applied, and we dont affect any children.

```suggestion
.accordion-header__actions {
```

---

Agreed 😊
If we assume the use of fragment as input, this works